Tools/cases_generator/tier2_generator.py

```diff
@@ -149,6 +174,9 @@ def generate_tier2(
     for name, uop in analysis.uops.items():
         if uop.properties.tier_one_only:
             continue
+        if uop.properties.oparg_and_1:
+            out.emit(f"/* {uop.name} is split on (oparg & 1) */\n\n")
```

Comment:
This comment appears a bit late in the output (after the `_1` variant).

Reply:
I don't think we need to be worrying about the order of the generated code. No one will be reading it top to bottom.